src/components/ErrorPage.jsx

```diff
@@ -1,46 +1,51 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import { Button } from '@edx/paragon';
 import { FormattedMessage } from 'react-intl';
 import { Link } from 'react-router-dom';
 
-import apiClient from '../config/apiClient';
-
-export default class ErrorPage extends Component {
-  componentDidMount() {}
-
-  render() {
-    const { username } = apiClient.getAuthenticationState().authentication;
```

Comment:
Since this authentication state is in redux, I removed this dependency on apiClient and just got it from there instead (by connecting this component).